dozer-ingestion/aerospike/src/connector.rs

```diff
@@ -276,13 +314,15 @@ async fn event_request_handler(
     let event = json.into_inner();
     let state = data.into_inner();
 
+    debug!("Event data: {:?}", event);
```

Comment:
Logging all events under `debug` is a bit verbose. Let's either make this `trace` or not log it at all